test/cli_test.py

```diff
@@ -111,7 +112,7 @@ def test_add_move_remove_album(self):
 
         by_orig_year_path = self.lib_path(b'by-year/1982/Thriller/track 1.mp3')
         self.assertIsNotFile(by_year_path)
-        self.assertSymlink(by_orig_year_path, target_path)
+        self.assertSymlink(by_orig_year_path, target_path, True)
```

Comment:
Could you add a test case for relative links?

Reply:
I added test cases for:
- Undefined behaviour
- Defined behaviour, linktype = absolute
- Defined behaviour, linktype = relative
- Invalid linktype